etc/tendrl/monitoring-integration/monitoring-integration.conf.yaml.sample

```diff
@@ -29,6 +29,10 @@ datasource_name: 'Graphite'
 
 datasource_host: 127.0.0.1
 
-access: direct
+datasource_port: 10080
+
+carbon_port: 2003
+
```

Comment:
Add help comments for each item, example,  access: proxy (I am not able to understand the purpose of flag without help comment)